games-fps/shockolate/shockolate-20180924.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="System Shock, but cross platform!"
+DESCRIPTION="Based on the source code for PowerPC released by Night Dive Studios, Incorporated.\
+You need contents of data folder from the original game copied into res/data folder."
+HOMEPAGE="https://github.com/Interrupt/systemshock"
+COPYRIGHT="2018 Interrupt"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/Interrupt/systemshock/archive/1b1c664188ce736a343913de32afe627e48e7bbb.zip"
+CHECKSUM_SHA256="29685f42d4dc0e1b6a608ac61ce8f09fdce379784af73400fb9cd64aa9b6d51d"
+SOURCE_DIR="systemshock-1b1c664188ce736a343913de32afe627e48e7bbb"
```

Comment:
By convention we use `srcGitRev` for storing git hash. Also, the `.tar.gz` archive is much smaller, so please use that instead.

Per [Guidelines#Archive name](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name), it's required that the archive is in the `<portName>-<portVersion>.<extension>` format. This can be done via `SOURCE_FILENAME`.

```diff
 REVISION="1"
+srcGitRev="1b1c664188ce736a343913de32afe627e48e7bbb"
-SOURCE_URI="https://github.com/Interrupt/systemshock/archive/1b1c664188ce736a343913de32afe627e48e7bbb.zip"
+SOURCE_URI="https://github.com/Interrupt/systemshock/archive/$srcGitRev.tar.gz"
 CHECKSUM_SHA256="29685f42d4dc0e1b6a608ac61ce8f09fdce379784af73400fb9cd64aa9b6d51d"
+SOURCE_FILENAME="systemshock-$portVersion.tar.gz"
-SOURCE_DIR="systemshock-1b1c664188ce736a343913de32afe627e48e7bbb"
+SOURCE_DIR="systemshock-$srcGitRev"
```

Please note that I didn't calculate the new checksum for the `.tar.gz` archive in this diff. You will have to do that yourself.